src/App.js

```diff
@@ -147,6 +148,7 @@ export class App extends React.Component {
               <PrivateRoute exact path='/add/tenant' component={Dashboard} />
               <PrivateRoute exact path='/add/property' component={AddProperty}/>
               <PrivateRoute exact path='/add/manager' component={Dashboard} />
+              <PrivateRoute exact path='/manage/tenants' component={Tenants} />
```

Comment:
Hmm there's already a /manage/tenants on line 154 here. Technically this one will get hit first, but we should remove the duplicate